sal_interfaces/CBP/CBP_Commands.xml

```diff
@@ -24,35 +24,23 @@
         <Subsystem>CBP</Subsystem>
         <Version>3.8</Version>
         <Author>ecoughlin</Author>
-        <EFDB_Topic>CBP_command_moveAzimuth</EFDB_Topic>
-        <Alias>moveAzimuth</Alias>
+        <EFDB_Topic>CBP_command_move</EFDB_Topic>
+        <Alias>move</Alias>
         <Device></Device>
         <Property></Property>
         <Action></Action>
         <Value></Value>
         <Explanation>http://sal.lsst.org</Explanation>
         <item>
             <EFDB_Name>azimuth</EFDB_Name>
-            <Description>The amount to move the azimuth by in absolute.</Description>
+            <Description>The amount to move in azimuth axis.</Description>
             <IDL_Type>float</IDL_Type>
             <Units>degree</Units>
             <Count>1</Count>
         </item>
-    </SALCommand>
-    <SALCommand>
-        <Subsystem>CBP</Subsystem>
-        <Version>3.8</Version>
-        <Author>ecoughlin</Author>
-        <EFDB_Topic>CBP_command_moveAltitude</EFDB_Topic>
-        <Alias>moveAltitude</Alias>
-        <Device></Device>
-        <Property></Property>
-        <Action></Action>
-        <Value></Value>
-        <Explanation>http://sal.lsst.org</Explanation>
         <item>
             <EFDB_Name>altitude</EFDB_Name>
-            <Description>The amount to move the altitude by in absolute.</Description>
+            <Description>The amount to move in altitude axis.</Description>
```

Comment:
Please use `elevation` not `altitude` to be consistent with the telescope mount controllers (ATMCS and ATMount), as well as the MT Dome.

Reply:
Also please consider listing elevation first, again to be consistent with ATMCS and ATMount.